addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp

```diff
@@ -0,0 +1,567 @@
+	magazines[] += {
+		LIST_11("CUP_1Rnd_HEDP_M203"),
+		LIST_2("1Rnd_Smoke_Grenade_shell")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+	primaryWeapon[] = {"CUP_arifle_M4A1_black"};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	vest[] = {"CUP_V_B_Eagle_SP-C_Corpsman"};
+	backpack[] = {"CUP_B_USMC_MOLLE_WDL"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"B_Carryall_cbr"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_smg_MP5A5"};
+	scope[] = {};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_9x19_MP5"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	items[] += {"ACE_MapTools"};
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_TL",
+		"CUP_V_B_MTV_TL"
+	};
+	magazines[] += {
+		LIST_2("CUP_1Rnd_StarFlare_Red_M203"),
+		LIST_2("CUP_1Rnd_SmokeRed_M203")
+	};
+	linkedItems[] += {
+		"ACE_Yardage450",
+		"ItemGPS"
+	};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_SL",
+		"CUP_V_B_MTV_PistolBlack"
+	};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] += {
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"CUP_H_USMC_CAP_PRR_DES"};
+	vest[] = {"CUP_V_B_Eagle_SPC_Officer"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	vest[] = {"CUP_V_B_Eagle_SPC_RTO"};
+	backpack[] = {"CUP_B_Kombat_Radio_Olive"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"hlc_m249_pip4"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_AR",
+		"CUP_V_B_MTV_MG"
+	};
+	backpack[] = {"CUP_B_AssaultPack_Coyote"};
+	magazines[] = {
+		LIST_3("hlc_200rnd_556x45_M_SAW"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"B_Kitbag_cbr"};
+	backpackItems[] = {
+		LIST_3("hlc_200rnd_556x45_M_SAW"),
+		"ACE_EntrenchingTool"
+	};
+	linkedItems[] += {"ACE_Yardage450"};
+};
+
+class rat : car {
+	displayName = "Rifleman (AT)";
+	vest[] = {"CUP_V_B_Eagle_SPC_AT"};
+	secondaryWeapon[] = {"CUP_launch_M136"};
+	backPack[] = {"B_Kitbag_cbr"};
+	magazines[] += {"CUP_launch_M136"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_DMR",
+		"CUP_V_B_MTV_Marksman"
+	};
+	primaryWeapon[] = {"CUP_srifle_Mk12SPR"};
+	scope[] = {"cup_optic_leupoldmk4_mrt_tan"};
+	attachment[] = {"cup_acc_anpeq_2_desert"};
+	bipod[] = {"cup_bipod_harris_1a2_l_blk"};
+	magazines[] = {
+		LIST_7("ACE_30Rnd_556x45_Stanag_Mk262_mag"),

```

Comment:
Pretty sure the Mk12SPR usually uses 20Rnd mags.
![bild](https://user-images.githubusercontent.com/14627848/88378731-dd08ed80-cda1-11ea-89ed-5e9733e59977.png)

Reply:
Operationally, the Mk12 SPRs were seen being used by standard 30 round magazines. This is due to a variety of factors but mostly because 30 round magazines were issued and in plenty supply and as such could be replaced easily if damaged, unlike the 20 rounders which were not as proliferate in the supply system.

For Arma, we only have 30 round Mk262 mags (the ammunition issued to Mk12 SPRs) so I made a compromise between how it's typically portrayed and gameplay purposes.